__fixtures__/example-todo-prerender/.gitignore

```diff
@@ -0,0 +1,14 @@
+.idea
```

Comment:
Yo, I think it's totally fine to add `prerender` to a new route on the existing `example-todo`

Not a train smash to add more fixtures, I just like `example-todo` to be everything an expectant redwood app would be.

Reply:
Sure, no probs :). I'm gonna add a whole load of more pages though ok?

Reply:
Totally OK, I think it's better to have a single good representation of a real working app than a bunch of them, since we want to test against all those cases.

Reply:
👍🏾

I'm going to see how far I can get with the existing ones, but given the use of modules like styled-components (which aren't installed in the fixture), I may need to have a separate fixture anyway.